examples/provisioner/windows.yaml

```diff
@@ -0,0 +1,57 @@
+# This example provisioner will provision instances running Windows Server OS
+
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: windows2019
+spec:
+  requirements:
+    - key: kubernetes.io/os
+      operator: In
+      values: ["windows"]
+    - key: node.kubernetes.io/windows-build
```

Comment:
`windows-build` is not needed in the Provisioner. It might confuse the users I think.
When selecting the `amiFamily`, this will be applied automatically by Karpenter.
Applicable to 2022 as well.

Reply:
Got it. Will make the change to remove.